### PR TITLE
Pin qt5 a little harder at compilation time while we build out 5.15.15

### PR DESCRIPTION
**Pull request by hmaarrfk**
@adrianinsaval has dome some amazing work on qt5-webengine see https://github.com/conda-forge/qt-webengine-feedstock/pull/48

However, I would like to remove some of the pressure from "building it out quickly" by pinning builds to 5.15.8 during the qt-webengine build out (it must be CFEP03).

Thanks for the consideration

cc: @conda-forge/qt-main

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@h-vetinari @conda-forge/core how do you feel about this? we have had alot of trouble finding a windows champion for Qt and it seems like @adrianinsaval is stepping up. This would be needed to ensure that new packages do not get build with the latest qt, and so they stay valid during the qt-webengine build out (which always takes time).

Thanks for the review!

**Review by h-vetinari — APPROVED** [review]
Sounds good to me. QT is pretty special in how hard it is to unblock new patch versions. Happy to see this progressing! 🙏

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -795,19 +795,19 @@ is_freethreading:
...
-  - 5.15
+  - 5.15.8
 qhull:
   - 2020.2
 qpdf:
   - 11
 qt:
-  - 5.15
+  - 5.15.8
 qt_main:
```
Comment (hmaarrfk) [inline recipe/conda_build_config.yaml:809]:
```suggestion
# qt-main is temporarily (as of Oct 25, 2024) pinned to 5.15.8 to help with the
# build out of the qt stack for 5.15.15
# https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5670
qt_main:
```

**Comment by hmaarrfk** [comment]
Ok i am going to add an ABI migration branch too so that it doesn't fall behind and cause weird solves.

**Comment by gillins** [comment]
Should `pyqt` have been pinned to 5.15.9 instead? 5.19.8 doesn't seem to exist (see https://github.com/conda-forge/qgis-feedstock/pull/467)?

**Comment by hmaarrfk** [comment]
ah yes, i mean i have not been following pyqt very closely it seems like you are doing "mismatched" versions there.

https://github.com/conda-forge/pyqt-feedstock/blob/main/recipe/meta.yaml#L3

i'll release the pyqt pin and revert it.

**Comment by adrianinsaval** [comment]
webengine is fully built now, is this still needed?

**Comment by hmaarrfk** [comment]
No. It can be reverted. 

If you want to make the PR that would be helpful

**Comment by adrianinsaval** [comment]
#6634

**Pull request merged**